ci/terraform/oidc/api-gateway.tf

```diff
@@ -853,6 +863,8 @@ data "aws_cloudformation_stack" "orch_frontend_stack" {
 locals {
   nlb_dns_name = length(data.aws_cloudformation_stack.orch_frontend_stack) > 0 ? data.aws_cloudformation_stack.orch_frontend_stack[0].outputs["OrchFrontendNlbDnsName"] : null
   nlb_arn      = length(data.aws_cloudformation_stack.orch_frontend_stack) > 0 ? data.aws_cloudformation_stack.orch_frontend_stack[0].outputs["OrchFrontendNlbArn"] : null
+
+  oidc_origin_cloaking_waf_export_name = var.environment == "sandpit" ? "dev-oidc-cloudfront-CloakingOriginWebACLArn" : "${var.environment}-oidc-cloudfront-CloakingOriginWebACLArn"
```

Comment:
I was planning on pulling out a local for the 'secure pipelines' environment (eg dev for sandpit, environment otherwise) which might be useful here (although I can always update when I get to that)
Also if someone tries to deploy oidc to authdev is it going to complain?

Reply:
I think so long as they don't turn on this feature it won't complain in authdev. Would be good to tidy with that local, but I'd like to get this in now

Reply:
Yeah totally fine, I can do that later